### DISCUSSION TITLE
PNPM Monorepo and 2 projects: only one is updated

### DISCUSSION THREAD

**Discussion by leonardofaria** [vercel/community#2345] [Help]
### Summary

I have a GitHub repository that contains a monorepo with PNPM. This monorepo has app A and app B.

I have in Vercel 2 projects, one for app A and one for app B.

When I push code to GitHub, only one of the apps is deployed: 

![Screenshot 2023-05-07 at 12 19 04 AM](https://user-images.githubusercontent.com/13719/236663635-29983a4e-0099-499c-aed1-b66d972acd0d.jpg)

I didn't find a way to troubleshoot this in Vercel UI. In fact, this commit `More messages` is not in Vercel, as if Vercel is not getting the most updated commit from `main`. The last commit listed in Vercel UI is `Try to fix telegram bot - part II`, which matches the green check. 

I already tried to disconnect/connect the repo again and nothing. Any ideas on how to solve this?

**Comment by amyegan** [comment]
🤔 Are you using the [Ignored Build Step](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel) field? That can auto-fail certain deployments based on configuration
